wp-content/themes/core/components/header/subheader/Subheader_Controller.php

```diff
@@ -50,8 +71,71 @@ public function get_title_args(): array {
 
 		return [
 			Text_Controller::TAG     => 'h1',
-			Text_Controller::CLASSES => [ 'page-title', 'h1' ],
+			Text_Controller::CLASSES => [ 'page-title', 'h1', 'c-subheader__title' ],
 			Text_Controller::CONTENT => $this->get_page_title(),
 		];
 	}
+
+		/**
+	 * @return string
+	 */
+	public function get_container_classes(): string {
+		return Markup_Utils::class_attribute( $this->container_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_media_classes(): string {
+		return Markup_Utils::class_attribute( $this->media_classes );
+	}
+
+	/**
+	 * @return string
+	 */
+	public function get_content_classes(): string {
+		return Markup_Utils::class_attribute( $this->content_classes );
+	}
+
+	public function get_image_args() {
+		if ( ! has_post_thumbnail() ) {
+			return [];
+		}
+
+		return [
+			Image_Controller::IMG_ID => (int) get_post_thumbnail_id(),
+			Image_Controller::AS_BG        => true,
```

Comment:
@luciadeveloper & @ryanurban thoughts on making this an actual figure + image and using content-fit rather than a background image?

Reply:
Sure, it'd be great to do that going forward with bgd image implementations (better for a11y, seo, etc.).

Reply:
I got rid of the background image and using _figure_ now.  It is all in the image component css file.

Reply:
-> Changes about this topic need some review